### PR TITLE
AB#67488- Added Seafarer Details component

### PR DESCRIPTION
**Pull request by tropical-turtle**
Added a Seafarer Details component

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                <RadzenAccordionItem Text="Tombstone summary">
+                    <div><b>First and Last name:</b></div>
+                    <div>@Data.FirstName  @Data.LastName</div><br>
+                    <div><b>Candidate document number:</b></div>
+                    <div>@Data.CDN</div><br>
+                    <div><b>Date of birth:</b></div>
+                    <div>@Data.DateOfBirth.ToShortDateString()</div><br>
+
+                    <RadzenButton Style="background-color: #F7F9FC;color: #1A202C;
+                        border: 1px solid #1A202C;margin: 0px;padding: 0 1rem;font-size: 14px;">View full profile</RadzenButton><br>
```
Comment (tc-tibo) [inline src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor:21]: Let's flag this now to enhance this in the future with CSS.

Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor:21]: Good point . Agreed. appreciated.

File: src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Components;
+using CSF.SRDashboard.Client.Models;
+
+namespace CSF.SRDashboard.Client.Components
+{
+    public partial class SeafarerDetails
+    {
+        [Parameter]
+        public SeafarerDetailsModel Data { get; set; }
```
Comment (tc-tibo) [inline src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs:9]:
```suggestion
        public SeafarerDetailsModel Model { get; set; }
```
To be consistent with `CSF.SRDashboard.Client.Components.RequestDetail`

Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs:9]: Good point.

File: src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    <br />
+                </RadzenAccordionItem>
+            </Items>
+        </RadzenAccordion>
+    </Body>
+</Card>
+
+@code {
+
+}
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor:64]: We can remove these three lines.
```suggestion
```

File: src/CSF.SRDashboard.Client/Models/RequestDetailsPageModel.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Models
+{
+    public class RequestDetailsPageModel
+    {
+        public RequestDetailsPageModel()
+        {
+        }
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Models/RequestDetailsPageModel.cs:12]: No need for default empty constructors
```suggestion
```

File: src/CSF.SRDashboard.Client/Models/RequestGridsModel.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Models
+{
+    public class RequestGridsModel
+    {
+        public RequestGridsModel()
+        {
+
+        }
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Models/RequestGridsModel.cs:13]: Same here default constructors are there by default and we don't need to define empty ones.
```suggestion
```

File: src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace CSF.SRDashboard.Client.Models
+{
+    public class SeafarerDetailsModel
+    {
+        //tombstone
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string CDN { get; set; }
+        public DateTime DateOfBirth { get; set; }
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs:14]: In the future, we may want to separate these into separate models.

File: src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+        //medical summary
+        public string FitnessStatus { get; set; }
+        public List<string> Limitations { get; set; }
+        public DateTime DateOfExpiry { get; set; }
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs:19]: See above comment.

File: src/CSF.SRDashboard.Client/Pages/_Host.cshtml [hunk 8]
```diff
@@ -30,8 +30,7 @@
     <link href="css/site.css" rel="stylesheet" />
     <link href="~/css/components/component-library.css" rel="stylesheet" />
     <link rel="stylesheet" href="_content/Radzen.Blazor/css/default-base.css">
-    <link href="~/css/Ninja.css" rel="stylesheet" />
-
+    <link href="~/css/Radzen.css" rel="stylesheet" />
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Pages/_Host.cshtml:33]: We should also remove Ninja.css from the wwwroot if it is no longer used.

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
A few changes, nothing big. A few comments were more suggestions for the future and don't necessarily need to be changed right now.

File: src/CSF.SRDashboard.Client/Pages/ServiceDetails.razor [hunk 9]
```diff
@@ -14,11 +16,15 @@
...
-                    
+                    <CSF.SRDashboard.Client.Components.RequestDetail Model="this.RequestDetailComponentModel" />
+
                     <br />
 
                     <CSF.SRDashboard.Client.Components.Attachments />
+
+                    <br />
+
+                    <CSF.SRDashboard.Client.Components.SeafarerDetails Data="requestDetailsPageData.SeafarerDetails" />
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Pages/ServiceDetails.razor:27]:
```suggestion
                    <CSF.SRDashboard.Client.Components.SeafarerDetails Model="requestDetailsPageData.SeafarerDetails" />
```

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]

File: src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs (hunk 2 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs]: Changed

File: src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor (hunk 3 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor:64]: Removed

File: src/CSF.SRDashboard.Client/Models/RequestGridsModel.cs (hunk 5 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Models/RequestGridsModel.cs:13]: Removed

File: src/CSF.SRDashboard.Client/Pages/_Host.cshtml (hunk 8 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Pages/_Host.cshtml:33]: It was renames.

File: src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs (hunk 7 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Models/SeafarerDetailsModel.cs:19]: Good point.

File: src/CSF.SRDashboard.Client/Models/RequestDetailsPageModel.cs (hunk 4 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Models/RequestDetailsPageModel.cs:12]: removed

**Review by AlexandreSanscartier — APPROVED** [review]
God work looks good to me

**Pull request merged**